packages/experience-builder-sdk/src/hooks/useExperienceBuilder.ts

```diff
@@ -23,7 +23,7 @@ type UseExperienceBuilderProps = {
 };
 
 /**
- * @deprecated This hook is deprecated. We re-designed the SDK to enable client or server side rendering
+ * @deprecated This hook is deprecated. Use fetchBySlug or fetchById instead
```

Comment:
I guess this will be a major version bump. How about we drop this file completely as part of the new release?